plugins/lime-plugin-pirania/src/piraniaApi.js

```diff
@@ -0,0 +1,59 @@
+import api from "utils/uhttpd.service";
+
+export function listVouchers() {
+	return api
+		.call("pirania", "list_vouchers", {})
+		.then((response) => response.vouchers.map((voucher) => {
+			const { is_active, expiration_date } = voucher;
+			let status = is_active ? "used" : "available";
+			if (expiration_date < Date.now()) status = "disabled";
```

Comment:
This isn't right. Confused about the right way to check the status. Tipos @spiccinini @germanferrero ?